build_config/README.md

```diff
@@ -3,6 +3,10 @@
 Customizing the build behavior of a package is done  by creating a `build.yaml`
 file, which describes your configuration.
 
+The full format is described in the
+[docs/build_yaml_format.md](../docs/build_yaml_format.md) file, while this
```

Comment:
[nit] this link won't work from the README on the pub site. Maybe add a full URL?

Reply:
gah! ok :(